### DISCUSSION TITLE
Welcome to monome.github.io Discussions!

### DISCUSSION THREAD

**Discussion by tehn** [monome/monome.github.io#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tehn** [comment]
reply!

**Comment by dndrks** [comment]
whaaaaaaaaaaaaat lol i *JUST* got a GoList notification ahead of this —
synchronicity!!

**Comment by tehn** [comment]
lol just testing
